collectd-unbound.go

```diff
@@ -16,20 +17,73 @@ import (
 
 var (
 	metrics = map[string]string{
```

Comment:
list of metrics taken from OpenBSD's unbound v1.9.4. The latest 1.10 might have more metrics than listed here